client/src/App.js

```diff
@@ -7,31 +7,17 @@ import Home from "./components/Home";
 import PreviousWorkoutsPage from "./components/PreviousWorkoutsPage";
 import CurrentWorkout from "./components/CurrentWorkout";
 import ExerciseSetsPage from "./components/ExerciseSetsPage";
+import LoginV2 from "./components/LoginV2.tsx";
+import NotFoundPage from "./components/NotFoundPage.tsx";
 
 function App() {
   const [user, setUser] = useState(null);
 
-  useEffect(() => {
-    fetch("/auth").then((r) => {
-      if (r.ok) {
-        r.json().then((user) => setUser(user));
-      } else {
-        setUser(null);
-      }
-    });
-  }, []);
-
-  if (!user)
-    return (
-      <div>
-        <Login setUser={setUser} />
-      </div>
-    );
-
   return (
     <div className="App">
-      <NavBar setUser={setUser} />
+      <NavBar setUser={setUser} user={user} />
```

Comment:
☝️ Now conditionally rendering this component on whether or not the user is logged in.